packages/client/e2e/credit.spec.ts

```diff
@@ -81,15 +83,15 @@ test.describe("Credit", () => {
         .first()
         .locator("div")
         .first()
+
       // Wait for first quote response
       await expect(firstQuote).not.toContainText("Awaiting response", {
         timeout: Timeout.LONG,
       })
 
       await firstQuote.hover()
-
-      await firstQuote.getByText(/Accept/).waitFor({state: "visible"})
-      await firstQuote.getByText(/Accept/).click()
+      await firstQuote.getByText(/Accept/).waitFor({ state: "visible" })
+      await firstQuote.getByText(/Accept/).click({ timeout: Timeout.AGGRESSIVE })
```

Comment:
Prior we wait for the accept button to be visible so we make the test fail after 5 seconds in case the click action didn't succeed